.github/workflows/client-ios.yml

```diff
@@ -47,6 +47,7 @@ jobs:
           fi
       - run: echo "::add-path::$(pwd)/bin"
       - run: expotools ios-generate-dynamic-macros
+      - run: expotools ios-generate-dynamic-macros
```

Comment:
```suggestion
```
Do we really need another invocation of this process here? 🤔

Reply:
It's a hotfix for https://github.com/expo/expo/issues/9547

Reply:
Maybe some comment then?
At first glace it seems as a additional unnecessary line 😉

Reply:
I intend to remove this line before merging the PR to fix one bug at at time, but thanks for reminding me to make it clear this is not part of the PR. 👍